metrics/storage/podmetrics/reststorage.go

```diff
@@ -108,7 +107,7 @@ func (m *MetricStorage) Get(ctx genericapirequest.Context, name string, opts *me
 		return &metrics.PodMetrics{}, errMsg
 	}
 	if pod == nil {
-		return &metrics.PodMetrics{}, errors.NewNotFound(api.Resource("Pod"), fmt.Sprintf("%v/%v", namespace, name))
+		return &metrics.PodMetrics{}, errors.NewNotFound(v1.Resource("pod"), fmt.Sprintf("%v/%v", namespace, name))
```

Comment:
resource names are plural and lower-case.

Reply:
done